opentech/static_src/src/app/src/components/EmptyPanel/index.js

```diff
@@ -27,6 +27,10 @@ export default class EmptyPanel extends React.Component {
                 {column === 'applications' &&
                     <p>No results found.</p>
                 }
+
+                {column === 'reviews' &&
+                    <p>No reviews found</p>
```

Comment:
This looks wrong... We should be using a new component for the `ReviewEmptyPanel` not passing a value in. Appreciate you are just copying what was done before. Look at it if you have time, but its ok for now,.